charts/rasa-x/templates/rasa-open-source-ingress.yaml

```diff
@@ -30,7 +32,36 @@ spec:
   {{- end }}
 {{- end }}
   rules:
-  {{- range .Values.ingress.hosts }}
+  
+{{- if semverCompare ">=1.19-0" .Capabilities.KubeVersion.GitVersion -}}
+    {{- range .Values.ingress.hosts }}
+  - host: {{ .host | quote }}
+    http:
+      paths:
+      - backend:
+          service:
+            name: {{ $fullName }}-{{ $.Values.rasa.versions.rasaProduction.serviceName }}
+            port: 
+              number: {{ $.Values.rasa.port }}
+        path: /core/(.*)
+        pathType: Prefix
+      - backend:
+          service:
+            name: {{ $fullName }}-{{ $.Values.rasa.versions.rasaProduction.serviceName }}
+            port: 
+              number: {{ $.Values.rasa.port }}
+        path: /webhooks/
+        pathType: Prefix
+      - backend:
+          service:
+            name: {{ $fullName }}-{{ $.Values.rasa.versions.rasaProduction.serviceName }}
+            port: 
+              number: {{ $.Values.rasa.port }}
+        path: /socket.io
+        pathType: Exact
```

Comment:
```suggestion
        pathType: Prefix
```

Reply:
I thought this might be the exact URL so kept exact. Prefix LGTM. Will make a change.

Reply:
done